web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -441,6 +441,7 @@ $.extend(fixmystreet.set_up, {
 
         args.latitude = $('input[name="latitude"]').val();
         args.longitude = $('input[name="longitude"]').val();
+        args.prefill_reports = $(this).data('prefill');
```

Comment:
I wonder why these aren't all just set in the args dict above?

Reply:
Ah, and this should be set after the getJSON because otherwise it's passed to the server in the next line.

Reply:
Lastly could put role/data in args, not sure why they're separate arguments to the hook.